recipe/meta.yaml

```diff
@@ -15,7 +15,7 @@ source:
     - py36-rtree-0.9.4.patch  # [win and py==36]
 
 build:
-  number: 0
+  number: 1
```

Comment:
Please increase the build number to `2` (`1` was used in PR #110).

Reply:
Done